src/Plugin/Action/Order/ExportOrderAction.php

```diff
@@ -50,4 +56,33 @@ public function handle(array $parameters): Response
 
         return Pdk::execute(PdkActions::GET_ORDER_DATA, ['orderIds' => $orderIds]);
     }
+
+    /**
+     * @param  \MyParcelNL\Pdk\Plugin\Collection\PdkOrderCollection $orderCollection
+     *
+     * @return \MyParcelNL\Pdk\Shipment\Collection\ShipmentCollection
+     */
+    private function getShipments(PdkOrderCollection $orderCollection): ShipmentCollection
+    {
+        return $orderCollection->generateShipments()
+            ->groupBy('orderId')
+            ->reduce(static function (ShipmentCollection $acc, ShipmentCollection $shipments) {
+                $lastShipment = $shipments->last();
+
+                if ($lastShipment->multiCollo) {
+                    $labelAmount         = $lastShipment->deliveryOptions->labelAmount;
+                    $offset              = $shipments->count() - $labelAmount;
+                    $multiColloShipments = $shipments->slice($offset, $labelAmount);
+
+                    $multiColloShipments->each(static function (Shipment $shipment) use ($acc) {
+                        $acc->push($shipment);
+                    });
+
+                    return $acc;
+                }
+
+                $acc->push($shipments->last());
```

Comment:
Is er geen scenario waarbij meerdere non-multicollo shipments in 1 request gemaakt worden, zoals nu wel kan vanuit de plugins? Of wordt dat elders opgevangen?